python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -217,6 +217,9 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
         ys = catPlot[self.config.axisLabels["y"]].values
         mags = catPlot[self.config.axisLabels["mag"]].values
 
+        if len(xs) == 0 or len(ys) == 0:
+            return fig
```

Comment:
I fail to see how the length of `xs` and `ys` could differ, but I guess it doesn't hurt to check both.